pennylane/math/single_dispatch.py

```diff
@@ -138,6 +152,29 @@ def _take_autograd(tensor, indices, axis=None):
 )
 
 
+def _ifft2_tf(a, s=None, axes=(-2, -1), norm=None):
+    if axes != (-2, -1):
+        raise ValueError(
+            "TensorFlow does not support passing axes; the ifft "
+            "will always be performed over the inner-most 2 dimensions."
+        )
+
+    if norm is not None:
+        raise ValueError("TensorFlow does not support the 'norm' keyword argument.")
+
+    # TensorFlow only supports FFT of complex tensors
+    if a.dtype not in [_i("tf").complex64, _i("tf").complex128]:
+        if a.dtype is _i("tf").float64:
+            a = _i("tf").cast(a, dtype=_i("tf").complex128)
+        else:
+            a = _i("tf").cast(a, dtype=_i("tf").complex64)
+
+    return _i("tf").signal.ifft2d(input=a)
+
+
+ar.register_function("tensorflow", "fft.ifft2", _ifft2_tf)

```

Comment:
Sometimes, the functionality differs between the different frameworks. Here, I am using `ar.register_function` to register a _custom_ wrapper around the TensorFlow `ifft2`, to account for:

- TensorFlow, unlike Torch and NumPy, only accepts complex tensor input
- TensorFlow does not accept an `axes` or `norm` argument